client/modules/IDE/components/Header/MobileNav.jsx

```diff
@@ -238,6 +238,7 @@ const MobileNav = () => {
         )}
       </Title>
       <Options>
+        {title === 'My Stuff' && <StuffMenu />}
```

Comment:
It is safer to look at `pageName` rather than `title` because eventually we will have a translation on `"My Stuff"`.

Reply:
yeah